### PR TITLE
Interactive-bots: Improve task descriptions

### PR DESCRIPTION
**Pull request by abhayKashyap03**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@abhayKashyap03, it looks like you've already signed the Dropbox CLA. Thanks!

File: tasks/interactive-bots.md [hunk 1]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
   is how Zulip knows the request is from an authorized user.
 
   You can also create a new bot for a user in settings - it will have its own
-  api key and email. Remember to subscribe it to the streams you want to use the
+  API key and email. Remember to subscribe it to the streams you want to use the
    bot in.
 
-* Create a `~/.zuliprc-local` file with credentials:
-
+* Create a `~/.zuliprc-local` file with credentials (`nano ~/.zuliprc-local`):
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: I wouldn't prefer nano as an editor

File: tasks/interactive-bots.md [hunk 2]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
   ```
   [api]
-  key=<api-key>
-  email=<email>
-  site=<dev-url>
+  key=<api-key> (Of the bot)
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: Hm... it might be confusing and students can paste parenthesis content as well. I'd remove it.

File: tasks/interactive-bots.md [hunk 3]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
+  email=<email> (Of the bot)
+  site=<dev-url> (localhost/Zulip server)
   ```
 
-  Replace the placeholder `<api-key>` in the example with an existing user API
-  key and placeholder `<email>` with the user email. The placeholder `<dev-url>` 
-  should point to your development environment URL. 
-  
-* Create a `followup` stream.
+* Create a `followup` stream on your Zulip server.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: and restore these info ;-)

File: tasks/interactive-bots.md (hunk 2 above)
Comment (abhayKashyap03) [inline tasks/interactive-bots.md]: ok

File: tasks/interactive-bots.md [hunk 4]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
 
-* Use "Manage Streams" to subscribe your bot to the following streams: devel, social, and followup
+* Use `Manage Streams` to subscribe your bot (if you created one) to the following streams: `devel`,
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: hm... not sure the info is needed.

File: tasks/interactive-bots.md (hunk 3 above)
Comment (abhayKashyap03) [inline tasks/interactive-bots.md]: "restore" what?

File: tasks/interactive-bots.md [hunk 5]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
+`social`, and `followup`.
 
-* Run the followup bot:
+* Run the followup bot outside of your VM:
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: if you used `localhost` as site param I don't think you can run bot outside the VM ;-)

Comment (abhayKashyap03) [inline tasks/interactive-bots.md]: That was already there

File: tasks/interactive-bots.md (hunk 4 above)
Comment (abhayKashyap03) [inline tasks/interactive-bots.md]: I didn't change anything there.

File: tasks/interactive-bots.md (hunk 5 above)
Comment (abhayKashyap03) [inline tasks/interactive-bots.md]: I didn't change anything here either.

File: tasks/interactive-bots.md [hunk 6]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
   b. checking that the messages showed up in the `followup` stream, prepended by
   the sender email
 
-* Take screenshots showing that the bot is working, i.e. your terminal window
-with the bot working, messages sent by the bot in the followup stream. Add the
-screenshots to `interactive-bots/followup/<username>/`.
+* Take screenshots showing that the bot is working. Make sure to take screenshots
+of:
+
+ - the output in your terminal window with the bot running, including the command you used to
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: this line is too long (79 chars max) please ensure that every line is shorter than 80 chars

File: tasks/interactive-bots.md [hunk 7]
```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
...
+  # for own links!
+  ```
 
-* Edit the `triage_message`:
+
+* Edit the if statements of `triage_message`:
 
   ```
-  if message['display_recipient'] == 'links':
+  if message['display_recipient'] == 'followup':
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: in task B we describe links bot - why did you change it?

File: tasks/interactive-bots.md [hunk 8]
```diff
@@ -152,9 +172,9 @@ messages to users.
                                          'from %s:' % (original_sender,))
   ```
 
+
 * Create a links bot in settings for an existing user. Use its credentials in
-`~/.zuliprc-local`. Remember to subscribe the bot to the streams you want to use
- it in.
+`~/.zuliprc-local`. Subscribe the bot to `devel` and `social` streams.
```
Comment (TomaszKolek) [inline tasks/interactive-bots.md]: want to use was better I think

**Comment by abhayKashyap03** [comment]
There are some files in the PR and I have no idea how they got here.
For example, the screenshots and the notes.

**Comment by TomaszKolek** [comment]
Not super sure about task description but I think the task was only about improving the description not about making bot ;-) (Why did you add screenshots etc?)
Please also fix the conflict.

**Comment by abhayKashyap03** [comment]
I have no idea how those screenshots and notes came to this PR.
I didn't add them.

**Comment by abhayKashyap03** [comment]
And I'm not able to remove them.
I'll create a new PR.

**Comment by abhayKashyap03** [comment]
I'll close this PR and open a new one, because I'm not able to remove those files.

**Comment by trueskawka** [comment]
Another PR for this was opened at #428

File: tasks/interactive-bots.md (hunk 5 above)
Comment (trueskawka) [inline tasks/interactive-bots.md]: @TomaszKolek you can, that's intentional.

Wonder why this showed as change in this PR.
